cpp/ql/src/experimental/Security/CWE/CWE-295/CurlSSL.ql

```diff
@@ -0,0 +1,39 @@
+/**
+ * @name Disabled certifcate verification
+ * @description Disabling SSL certificate verification of host or peer could expose the communication to man-in-the-middle(MITM) attacks.
+ * @kind problem
+ * @problem.severity warning
+ * @id cpp/curl-disabled-ssl
+ * @tags security
+ *       external/cwe/cwe-295
+ */
+
+import cpp
+import semmle.code.cpp.dataflow.new.TaintTracking
+
+/** Models the `curl_easy_setopt` function call */
+private class CurlSetOptCall extends FunctionCall {
+  CurlSetOptCall() {
+    exists(FunctionCall fc, Function f |
+      f.hasGlobalName("curl_easy_setopt") and
```

Comment:
Hi @porcupineyhairs , this is why you're not getting any results on tests:
```suggestion
      f.hasGlobalOrStdName("curl_easy_setopt") and
```
You'll need to update the `.expected` file as well of course.  Other than that the test LGTM now.